app/__init__.py

```diff
@@ -34,6 +31,16 @@ def create_app(config_name):
         search_api_client=search_api_client
     )
 
+    for framework_data in data_api_client.find_frameworks().get('frameworks'):
```

Comment:
Interesting issue here - we will need to trigger a re-start of the application server (probably by re-deploying the app without any code changes) to load any new frameworks that are created in future.

This isn't the end of the world: previously we needed to deploy a new version of the app with the manifests explicitly loaded.

But it could be a tricky gotcha at some point.